### DISCUSSION TITLE
Attaching emitters generated by json

### DISCUSSION THREAD

**Discussion by JudeOsborn** [creativelifeform/three-nebula#177] [Q&A]
It's not clear to me how to move the entire particle systems around after they have been loaded via JSON. The system is not an Object3D object and I don't see any methods or properties that would allow me to change the position or add the entire system to a group (or Object3D) in real time.

For example,  imagine the particle system and its emitter in [this code example](https://codesandbox.io/s/three-nebula-quickstart-kz6uv?file=/src/index.js:514-525). How would I connect it to a cube so it moves whenever the cube moves?

**Comment by rohan-deshpande** [comment]
Instead of using a `Scene` as the container, can you try using any other kind of Object3D and move that around?

**Reply by Myziak** [reply]
OP ask about attaching particle system into object 3d. 
The ".add" method from THREE.JS doesn;t work, because system is not a object3D instance.

So... how we can attach particle to object?

**Reply by wSlava** [reply]
maybe it will be useful for someone else, from my code: 

`import Nebula, { SpriteRenderer } from "three-nebula";
....
let nebula = null;
let OBJ = new THREE.Group();

Nebula.fromJSONAsync(JSON, THREE).then(loaded => {
      const nebulaRenderer = new SpriteRenderer(OBJ, THREE);
      nebula = loaded.addRenderer(nebulaRenderer);
});

OBJ.rotateY(Math.PI / 2);
OBJ.scale.multiplyScalar(0.1);
scene.add(OBJ);

....

render(){

   nebula.update(); 
}
`

**Reply by QDMS** [reply]
How would I make particles follow a OBJ of a OBJ.. I have a transparent sphere that a rocket orbits around. The rocket is attached to sphere. when i do your code with the rocket the particles wont move with rocket. when I i add the sphere as a OBJ the and not the rocket the particles go to the sphere location
